pyasic/config/mining/presets.py

```diff
@@ -6,7 +6,9 @@ class MiningPreset(MinerConfigValue):
     power: int | None = None
     hashrate: int | None = None
     tuned: bool | None = None
-    modded_psu: bool = False
+    modded_psu: bool | None = None
```

Comment:
- Changed default for `modded_psu` to `None` since luxos doesn't have this parameter 
- Added frequency, voltage params since luxos includes those values in profiles